content_scripts/link_hints.coffee

```diff
@@ -155,10 +171,14 @@ LinkHints =
     # Check for attributes that make an element clickable regardless of its tagName.
     if (element.hasAttribute("onclick") or
         element.getAttribute("role")?.toLowerCase() in ["button", "link"] or
-        element.getAttribute("class")?.toLowerCase().indexOf("button") >= 0 or
-        element.getAttribute("contentEditable")?.toLowerCase() in ["", "contentEditable", "true"])
+        element.getAttribute("contentEditable")?.toLowerCase() in ["", "contentEditable", "true"] or
+        element.hasAttribute("vimium-has-onclick-listener") or
+        element.hasAttribute("vimium-has-delegated-onclick-listener"))
       isClickable = true
 
+    # Dispose the attribute so next time it is not included, in case listener has been removed
+    element.removeAttribute "vimium-has-delegated-onclick-listener"
```

Comment:
If you put `element.hasAttribute("vimium-has-delegated-onclick-listener")` into an `else if` condition, then we no longer have to call this on every element. IE:
```
        ...
        element.hasAttribute("vimium-has-onclick-listener"))
      isClickable = true
    else if element.hasAttribute("vimium-has-delegated-onclick-listener")
      isClickable = true
      element.removeAttribute "vimium-has-delegated-onclick-listener"
```